### PR TITLE
Show node/npm version numbers in `bokeh info`

### LINKED ISSUE #5649: Add nodejs and npmjs version numbers to `bokeh info`
Followup on issue #5645.

### PR DESCRIPTION
**Pull request by mattpap**
fixes #5649

### CODE REVIEW

File: bokeh/command/subcommands/info.py [hunk 1]
```diff
@@ -72,3 +73,5 @@ def invoke(self, args):
             print("IPython version     :  %s" % ipy_version)
             print("Bokeh version       :  %s" % __version__)
             print("BokehJS static path :  %s" % settings.bokehjsdir())
+            print("node.js version     :  %s" % nodejs_version() or "(not installed)")
+            print("npm version         :  %s" % npmjs_version() or "(not installed)")
```
Comment (bryevdv) [inline bokeh/command/subcommands/info.py]: can you make the wording consistent with `ipy_version` i.e. either `Not installed` or `(not installed)` Either is fine with me.

Comment (mattpap) [inline bokeh/command/subcommands/info.py]: @bryevdv, done.

**Comment by bryevdv** [comment]
This is printing like this on py3:
```
BokehJS static path :  /Users/bryan/work/bokeh/bokeh/server/static
node.js version     :  b'v6.3.1'
npm version         :  b'3.10.3'
```

**Comment by mattpap** [comment]
@bryevdv, done again.

**Comment by bryevdv** [comment]
This will need updating:

https://github.com/bokeh/bokeh/blob/9650b109b62d385d372eecc1f9d5a511bf4a3d49/bokeh/command/subcommands/tests/test_info.py#L29

**Comment by bryevdv** [comment]
I updated (wanted to see if new files not in the original PR could be updated from the web UI)

**Pull request merged**